optuna_dashboard/_config.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import annotations
 
+import argparse
+from dataclasses import dataclass
```

Comment:
This line is required to include the suggestion in https://github.com/optuna/optuna-dashboard/pull/1154/files#r2332995813.
```suggestion
from dataclasses import dataclass
from dataclasses import fields
```